masschallenge/django-accelerator
# [AC-5733] - Office Hours in Personalized Locations

**Changes Introduced in [AC-5733](https://masschallenge.atlassian.net/secure/RapidBoard.jspa?rapidView=1&modal=detail&selectedIssue=AC-5733):**
- Change the office hours locations such that they appear like this:
<img width="558" alt="office hours location" src="https://user-images.githubusercontent.com/5147457/41902935-4a76aff8-793d-11e8-9f37-c397f08feea3.png">
- change existing office hours data:
  - remote --> remote - see description
  - tel aviv --> other - see description

**Test:**
- checkout AC-5733 on django-accelerator and on accelerate, 
- run `make migrate`
- Go to some office hours (e.g. http://localhost:8181/admin/mc/mentorprogramofficehour/20110/?_changelist_filters=program__id__exact%3D26)
- See that the options are now changed.
